python/Ganga/Core/Sandbox/Sandbox.py

```diff
@@ -56,7 +56,15 @@ def createPackedInputSandbox(sandbox_files, inws, name):
 
 #
 # Future release with tarball module
-    with open(tgzfile, 'w') as this_tarfile:
+
+    if str(tgzfile).endswith('.gz') or  str(tgzfile).endswith('.tgz'):
+        file_format = 'gz'
+    elif  str(tgzfile).endswith('.bz2'):
+        file_format = 'bz2'
+    else:
+        file_format = ''
```

Comment:
It would be nice to be able to use the [`mimetypes`](https://docs.python.org/2.6/library/mimetypes.html) module to do this.